Erros with <angled> imports on promisekit.h file (version 1.7)

I've got this errors when I try to compile on release:

```
- ERROR | xcodebuild:  /Users/matteo/Library/Developer/Xcode/DerivedData/App-agyylfaavsayemdgzozvlpmubnef/Build/Products/Release-iphonesimulator/PromiseKit/PromiseKit.framework/Headers/PromiseKit.h:44:9: error: 'NSNotificationCenter+PromiseKit.h' file not found with <angled> include; use "quotes" instead
- ERROR | xcodebuild:  /Users/matteo/Library/Developer/Xcode/DerivedData/App-agyylfaavsayemdgzozvlpmubnef/Build/Products/Release-iphonesimulator/PromiseKit/PromiseKit.framework/Headers/PromiseKit.h:50:9: error: 'NSURLConnection+PromiseKit.h' file not found with <angled> include; use "quotes" instead
- ERROR | xcodebuild:  /Users/matteo/Library/Developer/Xcode/DerivedData/App-agyylfaavsayemdgzozvlpmubnef/Build/Products/Release-iphonesimulator/PromiseKit/PromiseKit.framework/Headers/PromiseKit.h:59:9: error: 'UIActionSheet+PromiseKit.h' file not found with <angled> include; use "quotes" instead
- ERROR | xcodebuild:  /Users/matteo/Library/Developer/Xcode/DerivedData/App-agyylfaavsayemdgzozvlpmubnef/Build/Products/Release-iphonesimulator/PromiseKit/PromiseKit.framework/Headers/PromiseKit.h:62:9: error: 'UIAlertView+PromiseKit.h' file not found with <angled> include; use "quotes" instead
- ERROR | xcodebuild:  /Users/matteo/Library/Developer/Xcode/DerivedData/App-agyylfaavsayemdgzozvlpmubnef/Build/Products/Release-iphonesimulator/PromiseKit/PromiseKit.framework/Headers/PromiseKit.h:65:9: error: 'UIView+PromiseKit.h' file not found with <angled> include; use "quotes" instead
- ERROR | xcodebuild:  /Users/matteo/Library/Developer/Xcode/DerivedData/App-agyylfaavsayemdgzozvlpmubnef/Build/Products/Release-iphonesimulator/PromiseKit/PromiseKit.framework/Headers/PromiseKit.h:68:9: error: 'UIViewController+PromiseKit.h' file not found with <angled> include; use "quotes" instead

```
If I replace \<angled\> imports by "quotes" imports everything works fine but I don't know if it is a good solution. Can someone help me please?

I'm using Xcode 7.3 and PromiseKit v 1.7 installed via cocoapods 1.0
Errors occur only when I try to compile on release.

```
- ERROR | xcodebuild: ../Release-iphonesimulator/PromiseKit/PromiseKit.framework/Headers/PromiseKit.h:44:9: error: 'NSNotificationCenter+PromiseKit.h' file not found with <angled> include; use "quotes" instead
- ERROR | xcodebuild:  ../Release-iphonesimulator/PromiseKit/PromiseKit.framework/Headers/PromiseKit.h:50:9: error: 'NSURLConnection+PromiseKit.h' file not found with <angled> include; use "quotes" instead
- ERROR | xcodebuild:  ../Release-iphonesimulator/PromiseKit/PromiseKit.framework/Headers/PromiseKit.h:59:9: error: 'UIActionSheet+PromiseKit.h' file not found with <angled> include; use "quotes" instead
- ERROR | xcodebuild: ../Release-iphonesimulator/PromiseKit/PromiseKit.framework/Headers/PromiseKit.h:62:9: error: 'UIAlertView+PromiseKit.h' file not found with <angled> include; use "quotes" instead
- ERROR | xcodebuild:  ../Release-iphonesimulator/PromiseKit/PromiseKit.framework/Headers/PromiseKit.h:65:9: error: 'UIView+PromiseKit.h' file not found with <angled> include; use "quotes" instead
- ERROR | xcodebuild: ../Release-iphonesimulator/PromiseKit/PromiseKit.framework/Headers/PromiseKit.h:68:9: error: 'UIViewController+PromiseKit.h' file not found with <angled> include; use "quotes" instead
```

Thanks!

--- a/objc/PromiseKit.h
+++ b/objc/PromiseKit.h
@@ -14,58 +14,58 @@
 #endif
 
 #ifdef PMK_ACACCOUNTSTORE
-#import <ACAccountStore+PromiseKit.h>
+#import <PromiseKit/ACAccountStore+PromiseKit.h>
 #endif
 #ifdef PMK_AVAUDIOSESSION
-#import <AVAudioSession+PromiseKit.h>
+#import <PromiseKit/AVAudioSession+PromiseKit.h>
 #endif
 #ifdef PMK_CLGEOCODER
-#import <CLGeocoder+PromiseKit.h>
+#import <PromiseKit/CLGeocoder+PromiseKit.h>
 #endif
 #ifdef PMK_CLLOCATIONMANAGER
-#import <CLLocationManager+PromiseKit.h>
+#import <PromiseKit/CLLocationManager+PromiseKit.h>
 #endif
 #ifdef PMK_CKCONTAINER
-#import <CKContainer+PromiseKit.h>
+#import <PromiseKit/CKContainer+PromiseKit.h>
 #endif
 #ifdef PMK_CKDATABASE
-#import <CKDatabase+PromiseKit.h>
+#import <PromiseKit/CKDatabase+PromiseKit.h>
 #endif
 #ifdef PMK_MKDIRECTIONS
-#import <MKDirections+PromiseKit.h>
+#import <PromiseKit/MKDirections+PromiseKit.h>
 #endif
 #ifdef PMK_MKMAPSNAPSHOTTER
-#import <MKMapSnapshotter+PromiseKit.h>
+#import <PromiseKit/MKMapSnapshotter+PromiseKit.h>
 #endif
 #ifdef PMK_NSFILEMANAGER
-#import <NSFileManager+PromiseKit.h>
+#import <PromiseKit/NSFileManager+PromiseKit.h>
 #endif
 #ifdef PMK_NSNOTIFICATIONCENTER
-#import <NSNotificationCenter+PromiseKit.h>
+#import <PromiseKit/NSNotificationCenter+PromiseKit.h>
 #endif
 #ifdef PMK_NSTASK
-#import <NSTask+PromiseKit.h>
+#import <PromiseKit/NSTask+PromiseKit.h>
 #endif
 #ifdef PMK_NSURLCONNECTION
-#import <NSURLConnection+PromiseKit.h>
+#import <PromiseKit/NSURLConnection+PromiseKit.h>
 #endif
 #ifdef PMK_SKREQUEST
-#import <SKRequest+PromiseKit.h>
+#import <PromiseKit/SKRequest+PromiseKit.h>
 #endif
 #ifdef PMK_SLREQUEST
-#import <SLRequest+PromiseKit.h>
+#import <PromiseKit/SLRequest+PromiseKit.h>
 #endif
 #ifdef PMK_UIACTIONSHEET
-#import <UIActionSheet+PromiseKit.h>
+#import <PromiseKit/UIActionSheet+PromiseKit.h>
 #endif
 #ifdef PMK_UIALERTVIEW
-#import <UIAlertView+PromiseKit.h>
+#import <PromiseKit/UIAlertView+PromiseKit.h>
 #endif
 #ifdef PMK_UIVIEW
-#import <UIView+PromiseKit.h>
+#import <PromiseKit/UIView+PromiseKit.h>
 #endif
 #ifdef PMK_UIVIEWCONTROLLER
-#import <UIViewController+PromiseKit.h>
+#import <PromiseKit/UIViewController+PromiseKit.h>
 #endif
 
 

--- a/objc/PromiseKit.h
+++ b/objc/PromiseKit.h
@@ -41,13 +41,13 @@
 #import <NSFileManager+PromiseKit.h>
 #endif
 #ifdef PMK_NSNOTIFICATIONCENTER
-#import <NSNotificationCenter+PromiseKit.h>
+#import "NSNotificationCenter+PromiseKit.h"
 #endif
 #ifdef PMK_NSTASK
 #import <NSTask+PromiseKit.h>
 #endif
 #ifdef PMK_NSURLCONNECTION
-#import <NSURLConnection+PromiseKit.h>
+#import "NSURLConnection+PromiseKit.h"
 #endif
 #ifdef PMK_SKREQUEST
 #import <SKRequest+PromiseKit.h>
@@ -56,16 +56,16 @@
 #import <SLRequest+PromiseKit.h>
 #endif
 #ifdef PMK_UIACTIONSHEET
-#import <UIActionSheet+PromiseKit.h>
+#import "UIActionSheet+PromiseKit.h"
 #endif
 #ifdef PMK_UIALERTVIEW
-#import <UIAlertView+PromiseKit.h>
+#import "UIAlertView+PromiseKit.h"
 #endif
 #ifdef PMK_UIVIEW
-#import <UIView+PromiseKit.h>
+#import "UIView+PromiseKit.h"
 #endif
 #ifdef PMK_UIVIEWCONTROLLER
-#import <UIViewController+PromiseKit.h>
+#import "UIViewController+PromiseKit.h"
 #endif
 
 

--- a/objc/PromiseKit.h
+++ b/objc/PromiseKit.h
@@ -41,13 +41,13 @@
 #import <NSFileManager+PromiseKit.h>
 #endif
 #ifdef PMK_NSNOTIFICATIONCENTER
-#import "NSNotificationCenter+PromiseKit.h"
+#import <NSNotificationCenter+PromiseKit.h>
 #endif
 #ifdef PMK_NSTASK
 #import <NSTask+PromiseKit.h>
 #endif
 #ifdef PMK_NSURLCONNECTION
-#import "NSURLConnection+PromiseKit.h"
+#import <NSURLConnection+PromiseKit.h>
 #endif
 #ifdef PMK_SKREQUEST
 #import <SKRequest+PromiseKit.h>
@@ -56,16 +56,16 @@
 #import <SLRequest+PromiseKit.h>
 #endif
 #ifdef PMK_UIACTIONSHEET
-#import "UIActionSheet+PromiseKit.h"
+#import <UIActionSheet+PromiseKit.h>
 #endif
 #ifdef PMK_UIALERTVIEW
-#import "UIAlertView+PromiseKit.h"
+#import <UIAlertView+PromiseKit.h>
 #endif
 #ifdef PMK_UIVIEW
-#import "UIView+PromiseKit.h"
+#import <UIView+PromiseKit.h>
 #endif
 #ifdef PMK_UIVIEWCONTROLLER
-#import "UIViewController+PromiseKit.h"
+#import <UIViewController+PromiseKit.h>
 #endif
 
 

